sdk/data/aztables/README.md

```diff
@@ -1,151 +1,343 @@
-# Azure Tables client library for Go
+# Azure Data Tables client library for Go
 
-Azure Table storage is a service that stores large amounts of structured NoSQL data in the cloud, providing
-a key/attribute store with a schema-less design.
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
+The Azure Data Tables client can be used to access Azure Storage or Cosmos accounts.
 
-Azure Cosmos DB provides a Table API for applications that are written for Azure Table storage that need premium capabilities like:
-
-- Turnkey global distribution.
-- Dedicated throughput worldwide.
-- Single-digit millisecond latencies at the 99th percentile.
-- Guaranteed high availability.
-- Automatic secondary indexing.
-
-The Azure Tables client library can seamlessly target either Azure Table storage or Azure Cosmos DB table service endpoints with no code changes.
+[Source code][source_code] | [Package][Tables_gopackage] | [API reference documentation][Tables_ref_docs]
 
 ## Getting started
-
-### Install the package
-Install the Azure Tables client library for Go :
+The Azure Data Tables SDK can access an Azure Storage or CosmosDB account.
 
 ### Prerequisites
-* An [Azure subscription][azure_sub].
-* An existing Azure storage account or Azure Cosmos DB database with Azure Table API specified.
-
-If you need to create either of these, you can use the [Azure CLI][azure_cli].
-
-#### Creating a storage account
+* Go versions 1.16 or higher
+* You must have an [Azure subscription][azure_subscription] and either
```

Comment:
Do we need to mention the subscription? It's pretty clear you'll end up needing one for either of the two things below?

Or is that just typical boilerplate and we'll all have it?

Reply:
It's typical boilerplate in other languages to point customers on where to go to create a subscription if they don't have one.